.github/workflows/lint.yml

```
@@ -2,25 +2,27 @@ name: Lint
 on:
   pull_request:
 
+permissions: {}
```

```suggestion
permissions:
  contents: read
```
Pretty sure you need this https://github.com/actions/checkout?tab=readme-ov-file#recommended-permissions